app/workers/identity/user_acceptable_verified_credential_counts_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Identity
+  class UserAcceptableVerifiedCredentialCountsJob
+    include Sidekiq::Worker
+
+    LOG_MESSAGE = '[UserAcceptableVerifiedCredentialCountsJob] - Ran'
+    PROVIDERS = %w[idme logingov dslogon mhv].freeze
+
+    def perform
+      build_total_verified_log_data
+      build_provider_verified_log_data
+
+      Rails.logger.info(LOG_MESSAGE, log_data)
+    end
+
+    private
+
+    def build_total_verified_log_data
```

Comment:
I'm gonna look more into the `build_provider_verified_log_data` part of this later today. but feedback for `build_total_verified_log_data`:
* I think it looks good for overall totals
* We probably will want additional queries that measure how many UserAVC values were added in the 'last week', 'last month', 'last year'
  * For example, how many AVC records were created in the last week: `UserAcceptableVerifiedCredential.where(acceptable_verified_credential_at: 1.week.ago..).count`

Reply:
(Basically the goal of the last week/month/year is to make it so that Datadog can keep its metrics, but we aren't reliant on yesterday's Sidekiq metrics job to run for today's information. I just want to make sure that if the job doesn't run for a few days for whatever reason, we still have accurate metrics)

Reply:
Ok, cool. That makes sense. I can add those to the totals.